tests/ui/imports/glob-conflict-cross-crate-3.rs

```diff
@@ -0,0 +1,15 @@
+// check-pass
+// aux-build:glob-conflict-cross-crate-2-extern.rs
+
+extern crate glob_conflict_cross_crate_2_extern;
+
+mod a {
+    pub type C = i32;
+}
+
+use glob_conflict_cross_crate_2_extern::*;
+use a::*;
+
+fn main() {
+    let _a: C = 1;
```

Comment:
```suggestion
    let _a: C = 1; // FIXME this use should be ambiguous
```
Could you add FIXME comments to all lines that get ERROR/WARN annotations in https://github.com/rust-lang/rust/pull/114682?
Including old tests like `tests/ui/imports/extern-with-ambiguous-3.rs`.